src/onnxruntime-genai.def

```diff
@@ -0,0 +1,16 @@
+LIBRARY
```

Comment:
On Linux you will need a similar thing. That's why the main ORT repo has a https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/gen_def.py file that reads a config file, symbols.txt, then generates different link config files on different platforms.

Reply:
Yes, I will add support for linux in a subsequent PR. This PR is mostly for the API and has been validated on windows so far.
Thanks for the information. I had to search quite a bit in ort to understand how it exported the functions. :)